test/helper.sh

```diff
@@ -0,0 +1,216 @@
+    gcloud compute instances create "${resource_name}" --zone="${ZONE}" --network="${resource_name}" --subnet="${resource_name}" --image-family="debian-11" --image-project="debian-cloud" --tags="allow-ssh"
+    gcloud container clusters create "${resource_name}" --zone="${ZONE}" --network="${resource_name}" --subnetwork="${resource_name}"
+    gcloud container clusters get-credentials "${resource_name}" --zone="${ZONE}"
+}
+
+###############################################################################
+# Cleanup the basic setup for recipe test.
+# Globals:
+#   ZONE
+#   SUBNET_REGION
+# Arguments:
+#     Name of the test. Used to generate suffix for resources.
+###############################################################################
+cleanup_gke_basic() {
+    local test_name
+    test_name="$1"
+    resource_suffix=$(get_hash "${test_name}")
+    resource_name="gke-net-recipes-${resource_suffix}"
+
+    gcloud container clusters delete "${resource_name}" --zone="${ZONE}" --quiet || true
+    gcloud compute instances delete "${resource_name}" --zone="${ZONE}" --quiet || true
+    gcloud compute networks subnets delete "${resource_name}" --region="${SUBNET_REGION}" --quiet || true
+
+    # Cleanup the firewalls associated with the network before deleting it.
+    firewalls=( $(gcloud compute firewall-rules list --filter="network=\"${resource_name}\"" --format="value(NAME)") )
+    for fw in "${firewalls[@]}"; do
+        echo "${fw}" | awk '{printf "gcloud compute firewall-rules delete %s --quiet\n", $1}' | bash || true
+    done
+
+    gcloud compute networks delete "${resource_name}" --quiet || true
+}
+
+###############################################################################
+# Network environment setup for internal load balancer so that the load
+# balancer proxies can be deployed.
+# Globals:
+#   None
+# Arguments:
+#   Name of the test, used to generate subnet and firewall suffix.
+###############################################################################
+setup_ilb() {
+    local test_name
+    test_name="$1"
+    resource_suffix=$(get_hash "${test_name}")
+    resource_name="gke-net-recipes-${resource_suffix}"
+
+    proxy_only_subnet="proxy-only-${resource_suffix}"
+    allow_proxy_firewall="allow-proxy-${resource_suffix}"
+    proxy_only_subnet_range="10.129.0.0/23"
+
+    gcloud compute networks subnets create "${proxy_only_subnet}" --region="${SUBNET_REGION}" --purpose="REGIONAL_MANAGED_PROXY" --role="ACTIVE" --network="${resource_name}" --range="${proxy_only_subnet_range}"
+    gcloud compute firewall-rules create "${allow_proxy_firewall}" --allow="TCP:80" --source-ranges="${proxy_only_subnet_range}" --network="${resource_name}"
+}
+
+###############################################################################
+# Validate load balancer traffic by sending a curl command.
+# Globals:
+#   ZONE
```

Comment:
I don't see ZONE referenced in this function.

Please clean up all the of the comments to match what is happening in the functions.